README.md

```diff
@@ -1,25 +1,77 @@
-# README
+# BudgetTK
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Budget app is a mobile application, that helps you where to manage your budget: we have a list of transactions associated with a category, so that we can see how much money you spent and on what.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Ruby on Rails
+- Postgresql
+- Bootstrap
 
-* System dependencies
+## Getting Started
 
-* Configuration
+**To get a local copy up and running follow these simple example steps:**
 
-* Database creation
+### Prerequisites
 
-* Database initialization
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Rails](https://gorails.com/)
 
-* How to run the test suite
+### Setup
 
-* Services (job queues, cache servers, search engines, etc.)
+- Make sure you have Ruby on Rails set up properly on your computer
 
-* Deployment instructions
+``` sh
+  git git@github.com:sja-thedude/Recipe-App.git
+```
 
-* ...
-# BudgetApp
+### Install
+
+```sh
+  bundle install
+```
+
+### Run
+
+```
+  rails db: create
+```
+
+```
+  rails db:migrate
+```
+
+```
+  rails s
+```
+
+### Test
+
+```sh
+  rspec
+```
+### Usage
+
+To start recipe app you need to first `signup`
+
+## Author
+
+👤 **GABRIEL NWACHUKWU**
+
+- GitHub: [@codePriest](https://github.com/gabrielcoder247)
+- LinkedIn: [@codepriest](https://www.linkedin.com/in/gabriel-nwachukwu-209613173/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+- Feel free to check the [issues page](https://github.com/faizi2500/recipe-app/issues).
+
+## Show your support
+
+- Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse for giving us this chance
```

Comment:
- [ ] Remember to give credits to the author of the original design  [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)